src/linear_algebra/free_module/finite/rank.lean

```diff
@@ -104,6 +104,25 @@ lemma finrank_matrix (m n : Type*) [fintype m] [fintype n] :
   finrank R (matrix m n R) = (card m) * (card n) :=
 by { simp [finrank] }
 
+variables {R M N}
+
+/-- Two finite modules are isomorphic if they have the same (finite) rank. -/
```

Comment:
These modules are free, right? The docstring is misleading and should be fixed I think. Similar below.

Reply:
Good catch